### PR TITLE
smpeg2 and SDL2_mixer

### PR DESCRIPTION
**Pull request by ayghor**

### CODE REVIEW

File: srcpkgs/smpeg2/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'smpeg2'
+pkgname=smpeg2
+version=2.0.0
+revision=1
+build_style=gnu-configure
+#configure_args="--with-sdl-prefix=${XBPS_CROSS_BASE}/usr --disable-gtk-player --disable-opengl-player"
+#make_build_args="LDFLAGS+=-lstdc++"
```
Comment (Vaelatern) [inline srcpkgs/smpeg2/template]: These are commented out. If they are not useful, then delete them

Comment (ayghor) [inline srcpkgs/smpeg2/template]: done

File: srcpkgs/SDL2_mixer/template [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'SDL2_mixer'
 pkgname=SDL2_mixer
 version=2.0.1
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args="--disable-static"
+#configure_args="--disable-static"
```
Comment (Vaelatern) [inline srcpkgs/SDL2_mixer/template]: Why not delete the line?

File: srcpkgs/smpeg2/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+makedepends="SDL2-devel"
+short_desc="SDL2 MPEG Player Library"
+homepage="https://icculus.org/smpeg/"
+license="LGPL-2.1"
+maintainer="rogi <rogi@skylittlesystem.org>"
+distfiles="https://www.libsdl.org/projects/smpeg/release/smpeg2-${version}.tar.gz"
+checksum=979a65b211744a44fa641a9b6e4d64e64a12ff703ae776bafe3c4c4cd85494b3
+
+smpeg2-devel_package() {
+	depends+=" smpeg2>=${version}_${revision}"
```
Comment (Vaelatern) [inline srcpkgs/smpeg2/template]: `s/depends+=" /depends="/`

Comment (ayghor) [inline srcpkgs/smpeg2/template]: Yes, I see now it makes more sense. This probably means teh older smpeg package has similar problem. Will fix tomorow morning.

File: srcpkgs/SDL2_mixer/template (hunk 2 above)
Comment (ayghor) [inline srcpkgs/SDL2_mixer/template]: Sorry for that one. In fact, I did fix it, but probably messed up rebasing or didn't push properly. Tomorow morning I'll send all teh changes.

File: srcpkgs/smpeg2/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+makedepends="SDL2-devel"
+short_desc="SDL2 MPEG Player Library"
+homepage="https://icculus.org/smpeg/"
+license="LGPL-2.1"
+maintainer="rogi <rogi@skylittlesystem.org>"
+distfiles="https://www.libsdl.org/projects/smpeg/release/smpeg2-${version}.tar.gz"
+checksum=979a65b211744a44fa641a9b6e4d64e64a12ff703ae776bafe3c4c4cd85494b3
+
+smpeg2-devel_package() {
+	depends="smpeg2>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/smpeg2/template]: `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"`

Comment (ayghor) [inline srcpkgs/smpeg2/template]: done

**Pull request merged**